web/sites/default/config/group.content_type.employee-group_node-notification.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 4e5abe63-4ef0-4c56-b4a0-00a02db2c73e
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.type.employee
+    - node.type.notification
+  module:
+    - gnode
+    - node
+id: employee-group_node-notification
+label: 'Open Group: Group node (Notification)'
```

Comment:
While you added the content type to the groups, no permissions were granted. So while the tests passed for a site admin, they would have failed as an authenticated user, group admin, group editor. Group admins should have create and edit. All users should be able to see the published notifications for open groups. Only members should see the restricted group notifications.

Reply:
I left some screen shots on the jira board story. Could you clarify what you mean by permissions?